index.bs

```diff
@@ -297,13 +305,13 @@ spec: webidl
           the UA receives <a>new information about the user's intent</a>.
         </li>
         <li>
-          Ask the user to choose one of the options or deny permission, and wait
-          for them to choose. If the calling algorithm specified extra
-          information to include in the prompt, include it.
+          Wait for the user to choose either one of the options or deny permission.
+          If the calling algorithm specified extra information to include in the
+          prompt, include it.
         </li>
         <li>
           If the user chose an option, return it; otherwise return {{"denied"}}.
-          Depending on the details of the user's interaction, the UA may also
+          Depending on the details of the user's interaction,
```

Comment:
We could use something like "If the user's interaction indicates they intend this choice to apply to other realms, then ..."?

We do need to let UAs vary on how they interpret choices here.